ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -205,10 +217,39 @@ Item {
     }
 
     ListView {
+        id: transactionListRoot
         anchors.topMargin: 20
         anchors.fill: parent
+        anchors.bottomMargin: 69
```

Comment:
I tested your branch and this margin doesn't seem to be respected. After fixing the extraButtons row, it should work correctly and you probably won't need this

Reply:
I removed it and used width and height instead to ensure the tab is displayed correctly. I.e `height: parent.height - extraButtons.height`